xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1601,6 +1666,20 @@ io_stats_dump_global_to_dict(xlator_t *this, struct ios_global_stats *stats,
             goto out;
         }
     }
+
+    /* Set the fop-hit-per-sec only for incremental stats */
+    if (total_fop_hits && (interval >= 0) && (sec > 0)) {
```

Comment:
Shouldn't we add this info even if `sec` is 0, as you do in the log ?

Reply:
done